lib/request-handler.js

```diff
@@ -142,12 +104,16 @@ module.exports = function processRequest(options, request, response) {
             }
 
             // Make resources early discoverable while processing HTML
-            const preloadAssets = headers.link
-                ? getAssetsToPreload(headers, request)
-                : '';
-            if (preloadAssets !== '') {
-                responseHeaders.link = preloadAssets;
-            }
+            const assetsToPreload = getFragmentAssetsToPreload(
+                headers,
+                request.headers
+            );
+
+            // Loader script must be preloaded before every fragment asset
+            const loaderScript = getLoaderScript(amdLoaderUrl, request.headers);
+            loaderScript !== '' && assetsToPreload.unshift(loaderScript);
```

Comment:
any reason not to use explicit `if () {}` here?

Reply:
To avoid the typecasting.